docs/topics/read-standard-input.md

```diff
@@ -0,0 +1,77 @@
+[//]: # (title: Read standard input)
+
+Use the `readln()` function to read data from the standard input. It reads the whole line as a string:
+
+```kotlin
+// Reads and stores the user input in a variable. For example: Hi there!
+val myInput = readln()
+
+println(myInput)
+// Hi there!
+
+// Reads and prints the user input without storing it in a variable. For example: Hi, Kotlin!
+println(readln())
+// Hi, Kotlin!
+```
+
+To work with data types other than strings, you can convert the input using conversion functions like `toInt()`, `toLong()`, `toDouble()`, `toFloat()`, or `toBoolean()` with dot syntax.
+It is possible to read multiple inputs of different data types and store each input in a variable:
+
+```kotlin
+// Converts the input from a string to an integer value. For example: 12
+val myNumber = readln().toInt()
+print(myNumber)
```

Comment:
I would also suggest using `println` on this page to print lines because otherwise reading next value will begin on the same line as the previous is output, e.g.:
![image](https://github.com/JetBrains/kotlin-web-site/assets/4257577/3af0f22a-ff8f-4d39-a4dc-3a06663e5422)